Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/PropertyEditorToolsSystem.cpp

```diff
@@ -27,6 +28,7 @@ namespace AzToolsFramework
     void PropertyEditorToolsSystem::RegisterDefaultHandlers()
     {
         PropertyEditorToolsSystemInterface::RegisterHandler<ContainerActionButtonHandler>();
+        PropertyEditorToolsSystemInterface::RegisterHandler<Prefab::OverrideIconHandler>();
```

Comment:
Similar to comments above. This might be more specific than this scope was intended for. I'm not aware of an existing way to register handlers from elsewhere though. That's something we might need, especially for handlers implemented in Gems.

Reply:
I think you can register them wherever you want, so long as they're not somehow used first. I'd put this somewhere prefab-specific, like when your adapter is initialized.

Reply:
Moved this registration to the constructor of PrefabAdapter